internal/db/migrations/000005_add_is_deleted_account_column_to_users.sql

```diff
@@ -0,0 +1,5 @@
+-- +migrate Up
```

Comment:
Could you please remove the word 'account' from the file name for consistency. Thanks.